docs/firestore/pagination.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Pagination
+description: Pagination example using Cloud Firestore.
+next: /functions/usage
+previous: /firestore/usage-with-flatlists
```

Comment:
In order for this to work, those two pages need to have changes as well to "splice" this page in, since they each have next/prev links.
I think /functions/usage will need it's prev link altered to no longer be /firestore/usage-with-flatlists (it should be /firestore/pagination now)
I think /firestore/usage/with-flatlists will need it's next link changed to firestore/pagination instead of /functions/usage